### DISCUSSION TITLE
Could you guys send me some ETH on the Rinkeby chain, please?

### DISCUSSION THREAD

**Discussion by mihighgh** [smartcontractkit/full-blockchain-solidity-course-py#227] [Q&A] [answered]
Hi everyone,

Could some please send me some ETH on the Rinkeby chain?

I've tried to post on Twitter but nothing comes through. Also, since yesterday, it looks like Rinkeby Faucet is down. I've used the Kovan chain as a substitute for most of the course, but at lesson 3 I can't use it anymore.

My address is: 0x4c9dDD1aE203BCfE2bCe9dc6a3E8Ffb1810d8249

Thank you very much in advance.

**Comment by cryptologr** [comment, marked as answer]
Hi mihighg:

I've been having the same issue for the last few days. I ended up requesting from https://app.mycrypto.com/faucet. They send 0.01 ETH at a time though. There may be a way to request more but I didn't bother yet hoping that I can get more from the original Rinkeby Faucet later. I just sent 0.01 ETH to your address now in the meantime. I hope it helps.

https://rinkeby.etherscan.io/tx/0x7ccbbf7c20a7de92a1b19efd86c7a5d2630839ad1a15f33e1f1fe402a870d0eb

**Reply by mihighgh** [reply]
Thanks a lot. It looks like today it's also working on my side. Yesterday I was receiving an error when I was introducing the captcha code.

**Comment by cromewar** [comment]
Also Chainlink manages their own faucets and you don't have yo send any tweets, you can use them with this link:
https://faucets.chain.link/
